From 75688b0ec51e1371dc3503c003f1cc0dc8667463 Mon Sep 17 00:00:00 2001
From: Charles Short <charles.short@windriver.com>
Date: Mon, 29 Nov 2021 20:50:16 +0000
Subject: [PATCH] Remove openstackclient

Remove build-Depends-Indep for python-openstackclient as it is
not being used and it is causing problems with the build-pkgs
tool

Signed-off-by: Charles Short <charles.short@windriver.com>

[Ported to python-barbicanclient v7.1.0-3 @ Epoxy]
Signed-off-by: jbarboza <joaopedro.barbozalioneza@windriver.com>
---
 debian/control | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/control b/debian/control
index 3bb61d5..6e4d9f2 100644
--- a/debian/control
+++ b/debian/control
@@ -18,7 +18,6 @@ Build-Depends-Indep:
  python3-fixtures,
  python3-hacking,
  python3-keystoneauth1 (>= 5.1.1),
- python3-openstackclient,
  python3-openstackdocstheme <!nodoc>,
  python3-oslo.config,
  python3-oslo.i18n,
-- 
2.34.1

